db/migrate/20160324174218_create_container_deployment_nodes.rb

```diff
@@ -0,0 +1,17 @@
+class CreateContainerDeploymentNodes < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_deployment_nodes do |t|
+      t.string :address
+      t.string :name
+      t.text :labels
+      t.belongs_to :container_deployment, :type => :bigint
+      t.belongs_to :vm,                   :type => :bigint
+      t.text :docker_storage_devices, :array => true, :default => []
+      t.string :docker_storage_data_size
```

Comment:
My comments seems to have been lost here as well

- I don't think this should have docker specific naming.  Perhaps just remove the `docker_` prefix, since `storage_devices` and `storage_data_size` seem fine as is.
- Why is size a string?  Typically we store a bigint and have it be in the base unit of bytes (See disks table, free_space column, as an example)